### PR TITLE
No takeover refresh

### PR DESCRIPTION
**Pull request by rich-nguyen**
This adds a check so that the existing ads refresh test don't reload ads on takeover pages. https://github.com/guardian/frontend/pull/19087

For this to work, those takeovers must be understood through `hasPageSkin` config, so adding a reminder to the admin page to target the correct `ng` ad unit.

### CODE REVIEW

**Comment by janua** [comment]
:+1:

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js [hunk 1]
```diff
@@ -4,13 +4,14 @@ import type { ImpressionViewableEvent } from 'commercial/types';
...
 import { enableLazyLoad } from 'commercial/modules/dfp/lazy-load';
+import config from 'lib/config';
 
 const shouldRefresh = (advert: Advert): ?boolean => {
     const sizeString = advert.size && advert.size.toString();
     const isFluid = sizeString === '0,0';
     const couldBeVideo = advert.id === 'dfp-ad--inline1';
 
-    return !isFluid && !couldBeVideo;
+    return !isFluid && !couldBeVideo && !config.page.hasPageSkin;
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: Nitpicking, but would the following be clearer given we're negating every variable in the predicate?

```
const sizeString = advert.size && advert.size.toString();
const isNotFluid = sizeString !== '0,0';
const isNotVideo = advert.id !== 'dfp-ad--inline1';
const isNotPageskin =  !config.page.hasPageSkin;		     
  		  
return isNotFluid && isNotVideo && isNotPageskin;
```

**Comment by LATaylor-guardian** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js (hunk 1 above)
Comment (rich-nguyen) [reply static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: I'm generally fine with these unless I have to apply de morgan's.

**Pull request merged**